src/app/qgsprojectproperties.cpp

```diff
@@ -2313,6 +2313,7 @@ void QgsProjectProperties::showHelp()
 
 void QgsProjectProperties::checkPageWidgetNameMap()
 {
+#if 0
```

Comment:
Can you undo this?

Reply:
Yes, forgot fo remove it (checkPageWidgetNameMap() runs into an assert on both of my development machines, making it impossible to open the project properties )

Reply:
What triggers the assert? The code is supposed to ensure that a static list of the tabs in the dialog matches what's actually present in the dialog, so I'm keen to know where this is failing

Reply:
The code probably does not consider the translation. The assert happens, if the german translation is used, but not if qgis is started with --lang en. Here is the console output:

QgisApp::projectPropertiesPagesMap() is outdated, please update. Missing Allgemein

Reply:
That's odd - from https://github.com/qgis/QGIS/blob/master/src/app/qgisapp.cpp#L10450 the map should contain the translation for "General". Could you debug and see what's actually contained in QgisApp::projectPropertiesPagesMap() and whether it does contain the translated names?

Reply:
I've had a similar issue https://issues.qgis.org/issues/20834 No time to investigate deeper unfortunately :/